sources/sqlserver/README.md

```diff
@@ -0,0 +1,70 @@
+# HarbourBridge: SQLServer-to-Spanner Evaluation and Migration
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
+using data from an existing PostgreSQL,MySQL,SQL server database. This README provides
```

Comment:
nit: maybe remove the PGSQL and MYSQL here since this is a sql server readme.

Reply:
PG and MySQL ref removed.